### DISCUSSION TITLE
Changing debug overlay keybinding

### DISCUSSION THREAD

**Discussion by groovyghoul** [wezterm/wezterm#5221] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Is there a way to disable the keybinding for Debug Overlay (or at least change it).

I have added the following to `wezterm.lua`

```lua
	{ key = "D", mods = "CTRL|ALT|SHIFT", action = wezterm.action.ShowDebugOverlay },
```

but `CTRL-SHIFT-L` still opens up the Wezterm Debug Overlay.

This is causing me issues in Neovim when trying to move from left to right in split windows.

Any suggestions would be most appreciated.

**Comment by groovyghoul** [comment]
I actually just figured out a way to do this:

I added `config.disable_default_key_bindings = true` to my config and then just added back in the "default" key bindings that I actually use. Not sure if this is the right way, but it seems to be doing the trick for me.

**Reply by wez** [reply, marked as answer]
You can assign ctrl-shift-L to DisableDefaultAssignment
https://wezfurlong.org/wezterm/config/lua/keyassignment/DisableDefaultAssignment.html

**Reply by groovyghoul** [reply]
Oh man...that's way better! Thanks, so much!
